packages/clerk-js/src/ui/components/UserProfile/__tests__/PasswordPage.test.tsx

```diff
@@ -11,6 +10,10 @@ const initConfig = createFixtures.config(f => {
   f.withUser({});
 });
 
+const changePasswordConfig = createFixtures.config(f => {
+  f.withUser({ password_enabled: true });
```

Comment:
I think there's also `f.withPassword()`

Reply:
`f.withPassword({ required: true, enabled: true })` does not seem to work tho, the tests will fails when they shouldn't